draft-ietf-rats-architecture.md

```diff
@@ -973,28 +973,40 @@ A more detailed discussion with examples appears in {{time-considerations}}.
 
 # Privacy Considerations
 
-The conveyance of Evidence and the resulting Attestation Results
-reveal a great deal of information about the internal state of a
-device.  In many cases, the whole point of the Attestation process is
-to provide reliable information about the type of the device and the
-firmware/software that the device is running.  This information might be
-particularly interesting to many attackers. For example, knowing that a device is
-running a weak version of firmware provides a way to aim attacks better.
-
-Evidence and Attestation Results data structures are expected to support
-integrity protection encoding (e.g., COSE, JOSE, X.509) and optionally might
-support confidentiality protection (e.g., COSE, JOSE).
-Therefore, if confidentiality protection is omitted or unavailable, the protocols
-that convey Evidence or Attestation Results are responsible for detailing what
-kinds of information are disclosed, and to whom they are exposed.
-
-Furthermore, because Evidence might contain sensitive information,
-Attesters are responsible for only sending such Evidence to trusted
-Verifiers.  Some Attesters might want a stronger level of assurance of
-the trustworthiness of a Verifier before sending Evidence to it.  In such cases,
-an Attester can first act as a Relying Party and ask for the Verifier's own
-Attestation Result, and appraising it just as a Relying Party would appraise
-an Attestation Result for any other purpose.
+Every claim in Attestation Evidence and Attestation Results is potentially PII (Personally Identifying Information) depending on the end-end use case of the attestation.
+Each deployment of attestation must take into account how each claim may or may not be used in a privacy violating manner by the relying parties receiving the attestation.
```

Comment:
```suggestion
Each deployment of remote attestation must take into account how each Claim may or may not be used in a privacy violating manner by Verifiers receiving Evidence or Relying Parties receiving Attestation Results.
```

Reply:
The text after "Verifiers" and "Relying Parties" is redundant / unnecessary since we explain in other sections the function of Verifiers and Relying Parties. Strike "receiving Evidence" and "receiving Attestation Results"